kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -197,6 +201,10 @@ public void prepare(final String tableName) {
             .columns(DATA_VALUE.column())
             .where(ROW_TYPE.relation().isEqualTo(RowType.DATA_ROW.literal()))
             .where(DATA_KEY.relation().isEqualTo(bindMarker(DATA_KEY.bind())))
+            .where(TIMESTAMP.relation().isGreaterThanOrEqualTo(bindMarker(TIMESTAMP.bind())))
```

Comment:
are you sure bloom filtering still gets used when doing scans? I think it should, but maybe good to double check.

Reply:
I ran a perf test over the weekend and it did pretty well, I'm pretty sure the bloom filtering is still applied as BF is applied just on the partition key to see whether the partition is present or not in an SST and we still have a full partition key clause. Unfortunately I don't see any docs indicating one way or another...